applications: nrf_desktop: Reduce log verbosity (#9697)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * Power module has been re-factored so that it uses upstream Zephyr INA23X sensor driver.

nRF Desktop
-----------

> * Set max compiled-in log level for Non-Volatile Storage (:kconfig:option:`CONFIG_NVS_LOG_LEVEL`) to warning.
>   Lowered log level for ``Identity x created`` log in :ref:`nrf_desktop_ble_bond` to debug.
>   The changes were done to avoid flooding logs.
> 

Reviewer: ```suggestion
* Changed:

   * Implemented adjustments to avoid flooding logs:

      * Set the max compiled-in log level to ``warning`` for the Non-Volatile Storage (:kconfig:option:`CONFIG_NVS_LOG_LEVEL`).
      * Lowered a log level to ``debug`` for the ``Identity x created`` log in the :ref:`nrf_desktop_ble_bond`.

```
